shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/widget/presentation/MainStudyPlanWidgetActionDispatcher.kt

```diff
@@ -76,12 +78,11 @@ internal class MainStudyPlanWidgetActionDispatcher(
             is InternalAction.CaptureSentryException -> {
                 sentryInteractor.captureException(action.throwable)
             }
-            is InternalAction.FetchPaymentAbility -> {
-                purchaseInteractor
-                    .canMakePayments()
-                    .getOrDefault(false)
-                    .let {
-                        onNewMessage(InternalMessage.FetchPaymentAbilityResult(it))
+            is InternalAction.FetchSubscriptionLimitType -> {
+                subscriptionInteractor
+                    .getSubscriptionLimitType()
```

Comment:
I don't know, but maybe it's necessary to force update `getSubscriptionLimitType(forceUpdate = true)` here?

StudyPlanWidgetReducer produces this action only on forceUpdate:
```kotlin
private fun getContentFetchActions(forceUpdate: Boolean): Set<Action> =
    setOfNotNull(
        InternalAction.FetchLearningActivitiesWithSections(),
        InternalAction.FetchProfile,
        InternalAction.UpdateCurrentStudyPlanState(forceUpdate),
        if (forceUpdate) InternalAction.FetchSubscriptionLimitType else null
    )
```

Reply:
@XanderZhu 
I've just realized, that we don't need FetchSubscriptionLimitType, because FetchLearningActivitiesWithSections fetch subscriptionWithLimitType. It's just a duplication.